src/system.h

```diff
@@ -47,8 +53,36 @@ struct Image {
     void Purge() {
         auto seconds_passed = wxGetLocalTime() - last_display;
         if (seconds_passed > 10) bm_display = wxNullBitmap;
-        // TODO: we could even purge bm_orig for even greater memory savings if we have png_data.
     }
+
+
+    vector<uint8_t> ConvertImageToVector(const wxImage &im, wxBitmapType bmt) {

```

Comment:
Vector is confusing, Buffer would be better

Reply:
Ok, I see your point. Solved with https://github.com/aardappel/treesheets/pull/367/commits/5e99efa6393f15b52067e2f51ef26904ab3b6a51